chain-signatures/node/src/cli.rs

```diff
@@ -376,6 +376,30 @@ mod tests {
 
     const ETH_CONTRACT_ADDRESS: &str = "f8bdC0612361a1E49a8E01423d4C0cFc5dF4791A";
 
+    #[test]
+    fn test_digest_blacksand() {
```

Comment:
We should eventually move this to a separate crate/repo and not have this here. The purpose of these is pretty much just a check for sanity and it's not directly a test that should be ran constantly on CI